docs/docs/10.5-clone-with-props.md

```diff
@@ -1,12 +1,12 @@
 ---
 id: clone-with-props
-title: Cloning ReactElement
+title: Cloning ReactElements
 permalink: clone-with-props.html
 prev: test-utils.html
 next: create-fragment.html
 ---
 
-In rare situations a element may want to change the props of a element that it doesn't own (like changing the `className` of a element passed as `this.props.children`). Other times it may want to make multiple copies of a element passed to it. `cloneWithProps()` makes this possible.
+You can use `cloneWithProps()` to copy an element and optionally set `props` on the copies. For example, you can set the `className`s of a elements accessible via `this.props.children`.
```

Comment:
The plan is to deprecate cloneWithProps. We'll probably kill it in the next version or the version after – if you want to just add a note at the top of the page saying to use cloneElement always, that would probably be best.